Service.Core/Network/Public/Include/Network/iAPI.hpp

```diff
@@ -45,125 +45,125 @@ namespace Network
         virtual std::optional<MessageResult> messageAllClients(std::shared_ptr<Connection> exceptionConnection, const Message& message) const = 0;
 
         /**
-         * @brief The method for checking of server ping.
-         * @param const std::shared_ptr<Connection>& client - for connect messages to the client or server.
+         * @brief The method for checking of service ping.
+         * @param const std::shared_ptr<Connection>& client - for connect messages to the client or service.
          * @param Message& message - for use the message header functionality.
          */
-        virtual std::optional<MessageResult> checkServerPing(std::shared_ptr<Connection> client, const Message& message) const = 0;
+        virtual std::optional<MessageResult> checkServicePing(std::shared_ptr<Connection> client, const Message& message) const = 0;
 
         /**
          * @brief The method for checking of receiving all messages by the client.
-         * @param const std::shared_ptr<Connection>& client - for connect messages to the client or server.
+         * @param const std::shared_ptr<Connection>& client - for connect messages to the client or service.
```

Comment:
here and further instead "for connect messages to the client or service" use "sender context"

Reply:
Done